Service/NotificationManager.php

```diff
@@ -0,0 +1,170 @@
+<?php
+  /**
+   * Created by groovili
+   * https://github.com/groovili
+   */
+  declare(strict_types = 1);
+
+  namespace Kopay\NotificationBundle\Service;
+
+  use Kopay\NotificationBundle\Entity\NotificationEmail;
+  use Kopay\NotificationBundle\Entity\NotificationMessageInterface;
+  use Kopay\NotificationBundle\Entity\NotificationPush;
+  use Symfony\Component\OptionsResolver\Exception\InvalidArgumentException;
+
+  /**
+   * Class NotificationManager
+   *
+   * @package Kopay\NotificationBundle\Service
+   */
+  class NotificationManager {
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    protected const NOTIFICATION_EMAIL = 0;
+
+    protected const NOTIFICATION_PUSH  = 1;
+
+    /**
+     * @var NotificationMessageInterface
+     */
+    protected $notification = null;
+
+    /**
+     * NotificationManager constructor.
+     *
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+      $this->configuration = $config;
```

Comment:
no validation? So, if I pass empty array here - what will happen? I would suggest to flattern this array and make each required key as an argument here.